### PR TITLE
ENT-4436: Add feature flag in Organization to display course skills in Publisher

### PR DESCRIPTION
**Pull request by sameenfatima78**
**JIRA:** 

This PR adds a feature flag in the Organization model which will be False by default. If enabled, the skills will show up in the Publisher in course edit page. 

- https://openedx.atlassian.net/browse/ENT-4332
- https://openedx.atlassian.net/browse/ENT-4436

**Screenshots:** 

Screenshot 2021-04-16 at 2 46 24 PM

### CODE REVIEW

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -205,6 +205,13 @@ class Organization(CachedMixin, TimeStampedModel):
             " generated.  When this flag is disabled, the key can be manually set."
         )
     )
+    enable_skills_in_publisher = models.BooleanField(
+        default=False,
+        verbose_name=_('Enable skills in Publisher'),
+        help_text=_(
+            "When this flag is enabled, the skill tags will show up in the course edit page in Publisher."
```
Comment (saleem-latif) [inline course_discovery/apps/course_metadata/models.py]: nit: use single quotes for strings. i.e. `'When this flag is enabled, the skill tags will show up in the course edit page in Publisher.'`

**Review by saleem-latif — APPROVED** [review]
One small comment, otherwise Looks Good 👍

**Review by waheedahmed — APPROVED** [review]
👍 once double-quotes changed to single.

**Comment by waheedahmed** [comment]
Actually, I have one question. Are you going to use this flag in some API endpoint or at the publisher frontend to determine whether you need to show the skills or not?

**Comment by sameenfatima78** [comment]
> Actually, I have one question. Are you going to use this flag in some API endpoint or at the publisher frontend to determine whether you need to show the skills or not?

@waheedahmed The latter one; I will be using it in the Publisher
Screenshot 2021-04-16 at 4 40 31 PM

File: course_discovery/apps/course_metadata/models.py [hunk 2]
```diff
@@ -205,6 +205,13 @@ class Organization(CachedMixin, TimeStampedModel):
             " generated.  When this flag is disabled, the key can be manually set."
         )
     )
+    enable_skills_in_publisher = models.BooleanField(
+        default=False,
+        verbose_name=_('Enable skills in Publisher'),
+        help_text=_(
+            'When this flag is enabled, the skill tags will show up in the course edit page in Publisher.'
+        )
+    )
```
Comment (muhammad-ammar) [inline course_discovery/apps/course_metadata/models.py:214]: @sameenfatima78 I think backend should not send skills to frontend if `enable_skills_in_publisher` is set `False`. Validations on frontend can be bypassed easily plus one can see the skills in the API response.

**Review by muhammad-ammar — APPROVED** [review]

File: course_discovery/apps/course_metadata/models.py (hunk 2 above)
Comment (sameenfatima78) [reply course_discovery/apps/course_metadata/models.py:214]: As discussed on call, this feature flag will be removed in the future and is only in place right now for testing on 2-3 partners so no action required here.

File: course_discovery/apps/course_metadata/models.py (hunk 1 above)
Comment (sameenfatima78) [reply course_discovery/apps/course_metadata/models.py]: Done

**Pull request merged**